### PR TITLE
post: Installing HMS Core in Android Studio Emulator

### PR DESCRIPTION
**Pull request by arthurpalves**
### What is this article about?
R: This is a quick guide with few and simple instructions on installing HMS Core in Android Studio Emulator. If you want to test your HMS implementation and you do not have a Huawei device, this is your best option.

### Checklist:

- [x] I am added to the list of authors correctly.
- [x] All images used in the post is in the correct folder (`assets/images/post/`).
- [x] My article has been reviewed internally before opening this Pull Request.
- [x] I have not modified someone else's previously published articles.
- [x] I double checked my changes locally at http://127.0.0.1:4000/ after running `bundle install && bundle exec jekyll serve`.
- [x] My changes generate no new errors

### CODE REVIEW

File: _posts/2023-09-29-installing-hms-core-in-as-emulator.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+**2.** Install the Huawei AppGallery APK
+
+You can either Google or install it from the [Huawei Consumer website](http://consumer.huawei.com/en/mobileservices/appgallery/).
+
+Download the AppGallery APK.
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_2.png" height="600"/>
+</div><br>
```
Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]: This fixed height is likely to cause issues, I'd recommend to use the figure component:
```suggestion
{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_2.png" %}
```

File: _posts/2023-09-29-installing-hms-core-in-as-emulator.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Allow installing unknown apps in Chrome browser and install the app.
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_2.1.png" height="600"/>
+</div><br>
```
Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]:
```suggestion
{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_2.1.png" %}
```

File: _posts/2023-09-29-installing-hms-core-in-as-emulator.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Open the AppGallery, you will be prompted to install HMS Core. Proceed with the installation.
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_3.png" height="600"/>
+</div><br>
+
+**4.** Allow the installation of unknown apps.
+
+Once the HMS Core has been downloaded, you will need to allow installing unknown apps from AppGallery to install it.
```
Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]:
```suggestion
Once the HMS Core has been downloaded, you need to allow installing unknown apps from AppGallery to install it.
```

File: _posts/2023-09-29-installing-hms-core-in-as-emulator.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_4.png" height="600"/>
+</div><br>
```
Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]:
```suggestion
{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_4.png" %}
```

File: _posts/2023-09-29-installing-hms-core-in-as-emulator.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+<img src="/assets/images/post/hms-core/hms_core_step_2.1.png" height="600"/>
+</div><br>
+
+**3.** Install HMS Core.
+
+Open the AppGallery, you will be prompted to install HMS Core. Proceed with the installation.
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_3.png" height="600"/>
+</div><br>
```
Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]:
```suggestion
{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_3.png" %}
```

File: _posts/2023-09-29-installing-hms-core-in-as-emulator.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+<img src="/assets/images/post/hms-core/hms_core_step_4.png" height="600"/>
+</div><br>
+
+**5.** Verify it all went well.
+
+Press “Done”, then press the back button, you will see the AppGallery login screen.
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_5.png" height="600"/>
+</div><br>
```
Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]:
```suggestion
{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_5.png" %}
```

File: _posts/2023-09-29-installing-hms-core-in-as-emulator.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+Verify that the HMS Core app is installed.
+
+<div style="margin:auto;text-align:center">
+<img src="/assets/images/post/hms-core/hms_core_step_6.png" height="600"/>
+</div><br>
```
Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]:
```suggestion
{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_6.png" %}
```

Comment (robinsonzimmermann) [inline _posts/2023-09-29-installing-hms-core-in-as-emulator.md]: Can we add some key notes or conclusion in the end of the article? Finishing the article with an image feels like it was cut short. At least something saying that you reached the end of the process

**Review by robinsonzimmermann — APPROVED** [review]

File: _posts/2023-09-30-installing-hms-core-in-as-emulator.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_5.png" max-height="600px" %}
+
+Verify that the HMS Core app is installed.
+
+{% include components/figure.html url="/assets/images/post/hms-core/hms_core_step_6.png" max-height="600px" %}
+
+# Conclusion
+
+Installing HMS Core in Android Studio Emulator is straight forward and now you can use its rich arrat of open device and cloud capabilities.
```
Comment (yash-kapila) [inline _posts/2023-09-30-installing-hms-core-in-as-emulator.md]: Typo? `arrat`

**Review by yash-kapila — APPROVED** [review]

**Pull request merged**